karrot/history/migrations/0010_auto_20191103_1957.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.6 on 2019-11-03 19:57
+
+from django.db import migrations
+import django_enumfield.db.fields
+import karrot.history.models
+
+
+class Migration(migrations.Migration):
```

Comment:
I shouldn't have committed this file.

Reply:
Resolved with https://github.com/yunity/karrot-backend/commit/cb8a4dd52f69bcb399ab8bf8991a202c17e5f3e0